fixup-mountpoints

```diff
@@ -1775,6 +1775,17 @@ case "$DEVICE" in
             -e 's block/bootdevice/by-name/modem sde7 ' \
             -e 's block/bootdevice/by-name/bluetooth sde16 ' \
             "$@"
+        ;;    
+        
+    "mata")
+        sed -i \
+            -e 's /dev/block/bootdevice/by-name/system /dev/block/platform/soc/1da4000.ufshc/by-name/system ' \
```

Comment:
Please use a specific partition and relative paths, like here:
```
-e 's block/bootdevice/by-name/system sde13 ' \
```

Reply:
I noticed that. I dropped the /dev/ from src, but aren't really sure what I need to do for the out, so it'll take me a moment.